kubernetes/utils/create_from_yaml.py

```diff
@@ -54,35 +55,46 @@ def create_from_yaml(
         processing of the request.
         Valid values are: - All: all dry run stages will be processed
     """
+    if path.exists(yaml_file):
+        with open(path.abspath(yaml_file)) as f:
+            yaml_file = f.read()
+    else:
+        yaml_file = io.StringIO(yaml_file)
```

Comment:
I'm not sure we need a stringio here. Also, rewriting the input to the function is kind of ambiguous.

Reply:
If I don't wrap the file with a io.StringIO, I will get later the following exception from
`yaml.safe_load_all`
```
E       ValueError: I/O operation on closed file.
```